src/js/script.js

```diff
@@ -1,37 +1,144 @@
 document.addEventListener('DOMContentLoaded', () => {
-  const taskInput = document.getElementById('taskInput');
-  const addTaskBtn = document.getElementById('addTaskBtn');
-  const taskList = document.getElementById('taskList');
+    const taskInput = document.getElementById('taskInput');
+    const addTaskBtn = document.getElementById('addTaskBtn');
+    const taskList = document.getElementById('taskList');
 
-  document.addEventListener('keypress', handleKeyPress);
+    // Load tasks from local storage
+    loadTasks();
 
-  addTaskBtn.addEventListener('click', handleAddTask);
+    addTaskBtn.addEventListener('click', addTask);
+    taskInput.addEventListener('keydown', function(event) {
+        if (event.key === "Enter") {
+            addTask();
+        }
+    });
+
+    function addTask() {
+        const taskText = taskInput.value.trim();
+        if (taskText === '') return;
 
-  function handleKeyPress(event) {
-    const key = event.key.toLowerCase();
-    if (key == 'enter') {
-      handleAddTask();
+        const taskItem = createTaskItem(taskText);
+        taskList.appendChild(taskItem);
+        taskInput.value = '';
+
+        saveTasks();
     }
-  }
 
-  function handleAddTask() {
-    const taskText = taskInput.value.trim();
-    if (taskText !== '') {
-      addTask(taskText);
-      taskInput.value = '';
+    function createTaskItem(taskText, subtasks = [], expanded = true) {
+        const taskItem = document.createElement('li');
+        taskItem.classList.add('task-item');
+
+        taskItem.innerHTML = `
+            <div class="task-controls">
+                <span>${taskText}</span>
```

Comment:
Can you rewrite this to not use the input into innerHTML? This allows the user to potentially write HTML as a task item and it will parse and generate the text as HTML instead of plain text which is a security issue.